src/vtp/web/api/main.py

```diff
@@ -22,19 +41,63 @@ async def root() -> dict:
     return {"version": "0.1.0"}
 
 
+# Endpoint #1
 @app.get("/vote/")
 async def get_vote_store_id() -> dict:
     """Create and store a unique Vote Store ID for each client"""
-    vote_store_id = str(random.randrange(100000, 999999))
+
+    svdo = SetupVtpDemoOperation(
+        election_data_dir=DEFAULT_ELECTION_DATA_DIR,
+        verbosity=VERBOSITY,
+    )
+    guid = svdo.run(
+        guid_client_store=True,
+    )
+
+    vote_store_id = guid
     # add VoteStoreID to list
     vote_store_ids.append(vote_store_id)
     return {"VoteStoreID": vote_store_id}
 
 
+# Endpoint #2
 @app.get("/vote/{vote_store_id}")
 async def get_empty_ballot(vote_store_id: str) -> dict:
     """Return an empty ballot for a given Vote Store ID"""
+
+    if vote_store_id in vote_store_ids:
+        # Cet a (the) blank ballot from the backend
+        cbo = CastBallotOperation(
+            guid=vote_store_id,
+            verbosity=VERBOSITY,
+        )
+        empty_ballot = cbo.run(
+            blank_ballot=DEFAULT_BLANK_BALLOT,
+            return_bb=True,
+        )
+        return {"blank-ballot": empty_ballot}
+    return {"error": "VoteStoreID not found"}
```

Comment:
That's a general question worth answering: do we favor starting API endpoints with the success case since it's likely to be common, or get all error checking out of the way first? Both are reasonable, but it will pay off for reviewing and debugging if we are consistent.

Reply:
My philosopy is this: if we can avoid executing code, we should do it. If we can bounce out of a function early because of an error state, let's bounce! Thus: error check first.

Reply:
I'm inclined to agree, and am 👍🏼 on that. I want to clarify something I failed to say explicitly though, which is the use case where the reverse makes sense: when the the vast majority of calls will succeed, and the error checking is complex enough to cause a performance hit.

I don't think this is likely to happen here and definitely shouldn't happen for the demo.